sdks/java/io/jms/pom.xml

```diff
@@ -59,6 +46,15 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
```

Comment:
Is this addition needed now? I'm guessing that it isn't, since it should be inherited from the parent.